module/meta.go

```diff
@@ -7,7 +7,8 @@ import (
 )
 
 type Meta struct {
-	Path string
+	Path  string
+	Files []string
```

Comment:
```suggestion
	Filenames []string
```
I think this would better communicate what we store inside the slice.

Reply:
Good idea